### ISSUE TITLE
SSL Support

### ISSUE THREAD

**Issue by kaos00723** [jasonrohrer/minorGems#3]
Network requests to websites that strictly use SSL only don't work.

You currently receive a "Failed to get/read sequence number" error when an OHOL server tries to write anything to the databases if the website automatically converts all HTTP requests to HTTPS by default.

**Comment by jasonrohrer** [comment]
Yeah... the OHOL codebase doesn't currently support SSL, which is a pretty complicated beast to integrate on all supported platforms.

This is why the OHOL protocol makes no assumptions about protocol messages being private.

You should configure your web server that's running OHOL to allow HTTP traffic.  In case this is too risky for you, you should probably install OHOL stuff on a separate web server from other stuff.
